### ISSUE TITLE
Allow different nfs options for different mounts

### ISSUE THREAD

**Issue by rbayliss** [adlogix/docker-machine-nfs#41]
Hey there,
It would be great if it was possible to use different NFS options for different shared folders.  For example, I want to share /Users with the standard "-alldirs -mapall=\$(id -u):\$(id -g)" setup, but I want to share another directory with "-alldirs -maproot=0".  This isn't currently possible, as all shared folders share the same options.  Any ideas?

**Comment by tonivdv** [comment]
Hey @rbayliss ,

I guess we could do something like this:

``` bash
docker-machine-nfs test --shared-folder="/var/www,-alldirs -maproot=0" --nfs-config="-alldirs -mapall=\$(id -u):\$(id -g)"
```

The --nfs-config would be applied to all shared-folder on which a override config is not given. The override would be given in the --shared-folder config after a e.g. ',' separation ?

**Comment by rbayliss** [comment]
Yep, that would definitely do it.  Love that it doesn't break BC either.

**Comment by jmreicha** [comment]
Does this work currently?  I'm facing an issue a similar issue to @rbayliss where I need to share one directory with root permissions and others with normal permissions.
